source/includes/webhooks/_scorecard_deleted.md

```diff
@@ -0,0 +1,30 @@
+## Scorecard deleted
+
+This web hook only fires when individual scorecards are destroyed.  This occurs when the Harvest API delete endpoint is used, or when the delete scorecard link is used in the application. This will not fire when a candidate or application is deleted. A candidate or application being deleted implies all their scorecards have been deleted with them.
```

Comment:
Looks like there is one on person:
```
has_many :scorecards_including_draft, :class_name => 'Scorecard', :dependent => :destroy
has_many :applications, :dependent => :destroy
```
So deleting a person will delete their scorecards, and I think deleting their last application might convert a person to a prospect and that might delete their scorecards. I can ask Tim or Moneil if you would like.